routes/persons.js

```diff
@@ -46,6 +68,31 @@ router.post('/create', middleware.continueIfLoggedIn, async (req, res) => {
 });
 
 
+router.post('/bulkCreate', async (req, res) => {
+  try {
+    let createdPersons = await models.Person.bulkCreate(req.body, {
+      individualHooks: true,
+      fields: Object.values(PERSON_FIELDS)
+    });
+
+    let persons = req.body.map((person, i) => {
```

Comment:
This is the only weird part about this. Perhaps we should match by email instead of trying to match by place in the array

Reply:
Not convinced that order is kept but looks like this example is depending on order as well.